packages/front/src/chess/chess.lib.ts

```diff
@@ -0,0 +1,114 @@
+import { BLACK, CHESS_BOARD_SIZE, WHITE } from './chess.constants'
+import {
+  EChessPiece,
+  TChessBoard,
+  TChessCastlingAbility,
+  TChessPieceColor,
+  TChessSquare,
+  TChessSquareIndex,
+  TChessState
+} from './chess.models'
+
+export function fenToChessPiece(fenPiece: EChessPiece): EChessPiece {
+  return fenPiece
+}
```

Comment:
So many pieces. It probably can't be helped here, but the naming makes the function feel a bit erratic.